inst/extdata/PA2022CH_fr_exec_summary/template.Rmd

```diff
@@ -509,7 +509,7 @@ Ce graphique indique l'exposition actuelle aux technologies à faible émission
 \end{minipage}%
 \begin{minipage}[t]{0.6\textwidth}
 
-Ce graphique affiche les scores d'alignement à la fois au niveau agrégé (barre la plus à gauche) et au niveau sectoriel. Le plus grand triangle bleu foncé indique le score obtenu par ce portefeuille et le plus petit triangle gris indique le résultat moyen obtenu par les pairs. La ligne horizontale délimite les scores alignés sur la trajectoire de 1,8 °C (ou plus) par rapport à ceux qui ne le sont pas. La trajectoire de 1,8°C correspond à l'objectif de l'Accord de Paris visant à limiter le réchauffement climatique à une hausse des températures « bien en dessous de 2°C ». Les scores indiquent le scénario sur lequel s'alignent les plans de production des entreprises bénéficiant d’investissements. Pour obtenir de plus amples informations, veuillez vous reporter aux deux représentations ci-dessous.
+Ce graphique affiche les scores d'alignement à la fois au niveau agrégé (barre la plus à gauche) et au niveau sectoriel. Le plus grand triangle bleu foncé indique le score obtenu par ce portefeuille et le plus petit triangle gris indique le résultat moyen obtenu par les pairs. La ligne horizontale délimite les scores alignés sur la trajectoire de 1,8 °C (ou plus) par rapport à ceux qui ne le sont pas. La trajectoire de 1,8°C correspond à l'objectif de l'Accord de Paris visant à limiter le réchauffement climatique à une hausse des températures \guillemetleft bien en dessous de 2°C\guillemetright. Les scores indiquent le scénario sur lequel s'alignent les plans de production des entreprises bénéficiant d’investissements. Pour obtenir de plus amples informations, veuillez vous reporter aux deux représentations ci-dessous.
```

Comment:
space after `\guillemetleft`

Reply:
not really sure if `\guillemetleft` needs to have a space after it or not?

Reply:
no space means it cannot tell where the command ends. the space seems needed and will be removed when rendered

Reply:
possible that you can close it with `\guillemetleft\`?

Reply:
fixed spacing using ` \ ` where appropriate